salt/utils/jinja.py

```diff
@@ -309,6 +309,26 @@ def to_bool(val):
     return False
 
 
+@jinja_filter('tojson')
+def tojson(val, indent=None):
```

Comment:
I would suggest to have a few of Unit Tests for this.

Reply:
I decided not to because A) Jinja 2.9+ has this filter built in, and B) the integration test I added already confirms that the content is dumped to JSON (before loading as YAML). I suppose I could add some tests just to be thorough.

Reply:
OK, unit test added.